src/component/textbox/Word.js

```diff
@@ -23,13 +23,13 @@ function Word(props) {
   }
 
   function setColor(i) {
-    if (props.input[i] === props.word[i] && props.idx === props.currentWord) {
+    if (props.input[i] === props.word[i] && props.wordLocation === props.currentWord) {
```

Comment:
I like to use the full name for index variables (`i` as `index`). I find it a little more readable doesn't cost that much time. I would echo the same for any "temporary" variables (like variables in loops, etc.).